public/css/user-profile.css

```diff
@@ -0,0 +1,263 @@
+* {
+    font-family: 'Satoshi', sans-serif;
+}
+
+body{
+    display: grid;
+    justify-content: center;
+}
+
+#Base-Container{
+    display: grid;
+    grid-template-rows: repeat(6, auto);
+    grid-template-columns: 1fr;
+    background-color: #FDF1BD;
+    place-items: center;
+    place-self: center;
+    justify-content: stretch;
+    border-radius: 25px;
+    margin-top: 50px;
+    text-align: center;
+    width: 95%;
+}
+
+#Base-Container-ProfilePicture-Container{
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+}
+
+#Base-Container-ProfilePicture{
+    width: 75px;
+    height: 75px;
+    border-radius: 55px;
+    background-color: #FFFFFF;
+    padding: 15px;
+    margin-top: -45px;
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+}
+
+#Base-Container-FullName-Container{
+    display: grid;
+    grid-column: 1 / span 1;
+    grid-row: 3 / span 1;
+    margin-left: 1em;
+}
+
+#Base-Container-Upload-Avatar-Centering-Container{
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    grid-column: 1 / span 1;
+    grid-row: 2 / span 1;
+    padding: 5px;
+    text-align: center;
+    height: 100%;
+    width: 100%;
+}
+
+#Upload-Avatar{
+    display: none;
+    text-align-last: center;
+    place-self: center;
+}
+
+#Base-Container-Email-Container{
+    align-items: flex-end;
+    grid-column: 1 / span 1;
+    grid-row: 4 / span 1;
+}
+
+#Base-Container-Password-Container{
+    grid-column: 1 / span 1;
+    grid-row: 5 / span 1;
+}
+
+#Base-Container-Update-Container{
+    grid-column: 1 / span 1;
+    grid-row: 6 / span 1;
+}
+
+#FullName, #Email, #Password{
+    font-size: 23px;
+}
+
+#FullName:disabled, #Email:disabled, #Password:disabled{
+    opacity: 50%;
+}
+#FullName:enabled, #Email:enabled, #Password:enabled{
+    opacity: 100%;
+}
+
+#Pen-1, #Pen-2, #Pen-3, #Pen-4{
+    user-select: none;
+    -moz-user-select: none;    
+    -ms-user-select: none;  
+    -webkit-user-select: none;   
+    background-color: #FFFFFF;  
+    box-shadow: 1.5px 1.5px 2px rgb(202, 187, 144);
+    border-radius: 25px;
+    padding: 5px;
+}
+
+#Pen-1:hover, #Pen-2:hover, #Pen-3:hover, #Pen-4:hover{
+    color: #c7c7c7;
```

Comment:
Put some commonly-used colours in CSS custom properties like:

```css
:root {
    --color-info: #c7c7c7;
}
p {
    color: var(--color-info);
} 
```
Makes it easier to keep track of the palette.